go-controller/pkg/ovn/namespace.go

```diff
@@ -10,6 +10,11 @@ import (
 	kapi "k8s.io/api/core/v1"
 )
 
+const (
+	// Annotation used to enable/disable multicast in the namespace
+	nsMulticastAnnotation = "netnamespace.network.openshift.io/multicast-enabled"
```

Comment:
@dceara can you change this to "k8s.ovn.org/multicast-enabled"?

Reply:
Done.